srcpkgs/kamoso/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'kamoso'
+pkgname=kamoso
+version=21.04.3
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
```

Comment:
We want to run tests.

Reply:
ok. I try to fix. `configure_args="-DBUILD_TESTING=OFF"` i deleted it